hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -73,19 +73,23 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
 
 public:
 
-  static QString userEmail() { return "ServiceHootApiDbReaderTest@hoottestcpp.org"; }
+  QString userEmail() { return QString("%1.ServiceHootApiDbReaderTest@hoottestcpp.org").arg(testName); }
 
   long mapId;
+  QString testName;
 
-  void setUp()
+  void setUpTest(const QString& test_name)
```

Comment:
This `setUp` function needed to not be called automatically by the CPPUNIT framework so that the unique test name could be passed in for creating the user.  Now it is called explicitly at the beginning of each test.